client/src/client.rs

```diff
@@ -443,6 +443,14 @@ pub trait RpcApi: Sized {
         self.call("getblockstats", &[height.into()])
     }
 
+    fn get_block_stats_fields(&self, height: u64, fields: &Vec<bitcoincore_rpc_json::BlockStatsFields>) -> Result<json::GetBlockStatsResultPartial> {
```

Comment:
Can use `fields: &[bitcoincore_rpc_json::BlockStatsFields]` so it also accepts slices. And `bitcoincore_rpc_json` is also aliased as just `json`.